driver/gl/raster_cache.go

```diff
@@ -0,0 +1,48 @@
+package gl
+
+import (
+	"image"
+	"os"
+	"sync"
+	"time"
+
+	"fyne.io/fyne"
+)
+
+type rasterInfo struct {
+	pix     *image.RGBA
+	w, h    int
+	alpha   float64
+	expires time.Time
+}
+
+var cacheDuration = (time.Minute * 5)
+var rasters = make(map[fyne.Resource]*rasterInfo)
+var rasterMutex sync.RWMutex
+
+func init() {
+	if t, err := time.ParseDuration(os.Getenv("FYNE_CACHE")); err == nil {
+		cacheDuration = t
+	}
+
+	janitor := func() {
```

Comment:
Semantics again, but why this instead of declaring a separate function outside of `init()`?

Reply:
Good Question.

Probably habit on my part, spinning up a maint goroutine that is essentially private to the function that uses it, where it would be dangerous to share that function outside of init()

Its a bit of a quick-n-dirty approach, package with global vars and all :)

Im sort of assuming that before long, this caching approach could / should be expanded to cache more than just SVG rasters ... so plenty of ops to really clean this up later I think.

But yeah - good call.